app/models/concerns/automated_compliance_utils.rb

```diff
@@ -1,13 +1,9 @@
 module AutomatedComplianceUtils
   extend ActiveSupport::Concern
 
-  #placheholder, will be removed once lookup_props is refactored
-  def published_requirement_template
-    RequirementTemplate.find_by(activity: activity, permit_type: permit_type)
-  end
-
-  def requirements_lookups
-    published_requirement_template&.lookup_props || {}
+  def requirements_lookups #a flattened list of requiements with the right information
+    # template_version&.lookup_props
+    template_version&.requirement_template&.lookup_props
```

Comment:
should we rely on the requirement_template? The current draft `requirement_template` state might be vastly different from what's there in the currently published `template_version` e.g. Removed and added a bunch of requirements that isn't in the current template version

Reply:
yes..I think we have separate card to remove thise lookup_props from the requirment template.  https://hous-bssb.atlassian.net/browse/BPHH-846